### PR TITLE
PR: Updated README for local Elastic Search, And Refactor Elastic Search Config

### PR DESCRIPTION
**Pull request by elraphty**
This PR: 

- Refactored the elastic search connection to be able connect to local and production
- It also updated the README file
- And added a config package to be used by other scrapper packages.

### CODE REVIEW

File: bitcoin.stackexchange.com/main.py [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 
-if __name__ == "__main__":
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
-    INDEX = os.getenv("INDEX")
+from config.conf import INDEX_NAME, DATA_DIR
 
-    BASE_DIR = os.getenv("DATA_DIR", ".")
+if __name__ == "__main__":
+    BASE_DIR = os.getenv(DATA_DIR, ".")
```
Comment (kouloumos) [inline bitcoin.stackexchange.com/main.py]: you can also import `BASE_DIR` from `config.conf`.

Additionally, if you wanted to do the config a bit more flexible you could structure it like [this one here](https://github.com/kouloumos/scraper/blob/scraperv2/scraper/config.py#L58-L124), but I don't consider it a must for now.

**Review by kouloumos — COMMENTED** [review]
looks good, I left a small nit for the import of all env variables from config

File: bitcoin.stackexchange.com/main.py (hunk 1 above)
Comment (elraphty) [reply bitcoin.stackexchange.com/main.py]: > you can also import `BASE_DIR` from `config.conf`.
> 
> Additionally, if you wanted to do the config a bit more flexible you could structure it like [this one here](https://github.com/kouloumos/scraper/blob/scraperv2/scraper/config.py#L58-L124), but I don't consider it a must for now.

Okay, thanks @kouloumos

**Comment by elraphty** [comment]
> looks good, I left a small nit for the import of all env variables from config

Thanks @kouloumos I have moved BASE_DIR to the config/conf.py

**Comment by kouloumos** [comment]
> [...] I have moved BASE_DIR to the config/conf.py

It seems that `BASE_DIR` == `DATA_DIR`, so we don't need both variables, right?

As I said, this is a nit, but we already started discussing it, so let's fix it and merge.

**Comment by elraphty** [comment]
> > [...] I have moved BASE_DIR to the config/conf.py
> 
> It seems that `BASE_DIR` == `DATA_DIR`, so we don't need both variables, right?
> 
> As I said, this is a nit, but we already started discussing it, so let's fix it and merge.

Oh!, I thought they served different purposes, making an update now.

File: mailing-list/package.json
Comment (kouloumos) [inline mailing-list/package.json:1]: I think that [main.js](https://github.com/bitcoinsearch/scraper/blob/master/mailing-list/main.js) contains code that we are not using anymore, as we transitioned to `main.py`, so maybe we can delete both `package.json`, `yarn.lock` and `main.js` as part of this PR?

Comment (elraphty) [reply mailing-list/package.json:1]: Sure, done.

**Review by kouloumos — APPROVED** [review]
Looks good!

**Pull request merged**